packages/url_launcher/url_launcher/README.md

```diff
@@ -164,3 +166,39 @@ If you do this for a URL of a page containing JavaScript, make sure to pass in
 `enableJavaScript: true`, or else the launch method will not work properly. On
 iOS, the default behavior is to open all web URLs within the app. Everything
 else is redirected to the app handler.
+
+## File scheme handling
+`file:` scheme can be used on desktop platforms: `macOS`, `Linux` and `Windows`.
+
+We recommend checking first whether the directory and/or file exists before calling the `launch`.
```

Comment:
Something can't be both a directory and a file, so "and/or" should just be "or".